spec/e2e/features/listings/listing.feature

```diff
@@ -7,5 +7,10 @@ Feature: Listings
       Then I should be redirected to the welcome page
 
     Scenario: Attempting to go to the Ownership listings page
-      Given I try to go to the Ownership listings page
+      Given I go to the Ownership listings page
       Then I should see available units
+
+    Scenario: Viewing and interacting with a listing page
```

Comment:
This is a very general name for this scenario, even though the scenario currently only contains a check for going to the listing page and interacting with the download application feature, because I envision that it will be expanded and divided as we progress in developing the listing-related parts of Ownership.